app/controllers/concerns/untrackable.rb

```diff
@@ -2,15 +2,15 @@ module Untrackable
   extend ActiveSupport::Concern
 
   included do
-    before_action :prevent_tracking
-    after_action :prevent_tracking
+    around_action :prevent_tracking
```

Comment:
Not blocking but couldn't you write this as 
```suggestion
    included { around_action :prevent_tracking }
```